frontend/local/src/modules/ehr/encounter.js

```diff
@@ -94,6 +94,12 @@ export default dispatch =>
             unit: "kg",
             formPath: "vitalSigns.weight.value"
         },
+        {
+            type: "dateTime",
+            ehrPath:
+                "/content[openEHR-EHR-COMPOSITION.encounter.v1]/context/other_context/items[openEHR-EHR-OBSERVATION.body_weight.v2]/context/start_time",
+            formPath: "vitalSigns.weight.timestamp"
+        },
```

Comment:
I made a few mistakes with these data points.
Quantity should be stored as 
```
/content[openEHR-EHR-COMPOSITION.encounter.v1]/context/other_context/items[openEHR-EHR-OBSERVATION.body_weight.v2]/data[at0002]/events[at0003]:0/data[at0001]/items[at0004]
```
and the time as
```
/content[openEHR-EHR-COMPOSITION.encounter.v1]/context/other_context/items[openEHR-EHR-OBSERVATION.body_weight.v2]/data[at0002]/events[at0003]:0/time
```